dmgmodify/main.go

```diff
@@ -29,7 +29,7 @@ func main() {
 		log.Fatal(err)
 	}
 
-	err = dmgmodify.WriteAttributionCode(dmgObj, os.Args[3], []byte(os.Args[4]))
+	err = dmgmodify.WriteAttributionCode(dmgObj, []byte(os.Args[4]))
```

Comment:
@bhearsum since we moved the sentinel value to WriteAttributionCode, it means we can't pass it in anymore.
Is this ok, or should we have 2 separate functions for handling custom sentinel values? (is there a use case for that ?)

Reply:
There's no known use case for other sentinel values at this time, so it's fine to ignore that case.